src/windows/wintab.c

```diff
@@ -252,12 +251,19 @@ void wintab_on_packet(struct wintab *ctx, MTY_Event *evt, const PACKET *pkt, MTY
 		if (double_click)
 			ctx->has_double_clicked = double_clicked = pressed;
 
-		if (pressed || prev_pressed)
-			evt->pen.flags |= 
-				i == 0 ? MTY_PEN_FLAG_TIP :
-				i == 1 ? MTY_PEN_FLAG_BARREL_1 :
-				i == 2 ? MTY_PEN_FLAG_BARREL_2 :
-				0;
+		if (ctx->override) {
+			if (pressed || prev_pressed)
+				evt->pen.flags |=
+					i == 0 ? MTY_PEN_FLAG_TIP :
+					i == 1 ? MTY_PEN_FLAG_BARREL_1 :
+					i == 2 ? MTY_PEN_FLAG_BARREL_2 :
+					0;
```

Comment:
This got changed in #44 what is the correct behavior?

Reply:
#45 has the correct behavior. I guess merging #44 first would be fine, as #45 is only surrounding #44 changes with an `if`/`else`.